C#/cs01/15.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+namespace Brainchild.Curriculum.Solutions
+{
+    // ArrayList like class IntList
+    // IntList should have two methods
+    // Add(int num)
+    // Get(int index)
+    // Count Property
+    public class IntList
+    {
+        public int[] integerArray = new int[1];
```

Comment:
variable names should not be reflecting C# names like int or float etc. you can name it like internalList or InternalArr. something which makes sense

Reply:
sure ,here on I will name it like that